contracts/README.md

```diff
@@ -0,0 +1 @@
+Write smth useful
```

Comment:
Just add a simple README (title + point to docs) using this example: https://github.com/smartcontractkit/chainlink-starknet/blob/develop/integration-tests/README.md

Will this README content be used in any way when publishing the package? I suppose not.

Reply:
It will be pasted in pkg info